src/Installer/ReleaseNotes.md

```diff
@@ -1,4 +1,12 @@
 ## DEV_VERSION_NUMBER: DEV_RELEASE_DATE
+* Chorus: Update documentation of Chorus Hub
+* FLExBridge: Improve developer documentation and update documentation of recent fixes
```

Comment:
this improvement isn't user-facing (but I'm not sure that this file is, either, is it?